src/pages/signUpView.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import TextInput from '../components/global/TextInput';
+import Button from '../components/global/Button';
+import { Link } from "react-router-dom";
+import SignUpBGImage from '../assets/images/SignupImage';
+
+export default function SignUpView( ) {
+    const [ firstName, setFirstName ] = useState("");
+    const [ lastName, setLastName ] = useState("");
+    const [ email, setEmail ] = useState("");
+    const [ password, setPassword ] = useState("");
+    const [ firstNameInfoText, setFirstNameInfoText ] = useState("");
+    const [ lastNameInfoText, setLastNameInfoText ] = useState("");
+    const [ emailInfoText, setEmailInfoText ] = useState("");
+    const [ passwordInfoText, setPasswordInfoText ] = useState("");
+
+    const signUp = () => {
+        // For debugging
+        console.log(`First name: ${firstName} \nLast name: ${lastName} \nEmail: ${email} \nPassword: ${password}`);
```

Comment:
Remove this console.log

Reply:
It'll be removed before it's merged into main. Jesse and I need to collate our work on this branch before we implement the authorisation stuff